sale_partcode_substitution/wizard/sale_partcode_replacement.py

```diff
@@ -53,5 +52,5 @@ def change_products_partcode(self):
                         )
                         vals = self._finalize_vals(vals, line, new_part)
                         line.write(vals)
-        sale._amount_all()  # TODO: Why is this needed now?
+        # sale._amount_all()  # TODO: Why is this needed now?
```

Comment:
Leave this in please, it is fixing a bug in production that we haven't proven doesn't require this second call yet.